### PR TITLE
LPS-67704 Regenerate.

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

File: modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingLocalServiceImpl.java [hunk 1]
```diff
@@ -1305,8 +1306,15 @@ protected void sendNotification(
...
-			
+			CalendarBooking parentCalendarBooking =
+				calendarBooking.getParentCalendarBooking();
+
+			CalendarResource calendarResource =
+				parentCalendarBooking.getCalendarResource();
+
+			Group group = groupLocalService.getGroup(
+				calendarResource.getGroupId());
+
```
Comment (brandizzi) [inline modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingLocalServiceImpl.java]: Hi @mosoni!

Just to note, it is possible to create events in calendars from outside the staging site from the staging site. E.g. if a user is in the staging site, clicks in "add event" but adds the event to the user's personal calendar, the notification should be sent. So, the condition to check is whether the parent event is from a staging group.

Said that, your fix is great, it was quite easy to complement it. Thanks!

Comment (mosoni) [inline modules/apps/forms-and-workflow/calendar/calendar-service/src/main/java/com/liferay/calendar/service/impl/CalendarBookingLocalServiceImpl.java]: Hi @brandizzi ,

Thanks for reviewing it and explaining me an actual behavior :)

**Comment by marcellustavares** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcellustavares** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/42920. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**